### PR TITLE
Fix list dir

### PR DESCRIPTION
**Pull request by oeway**
The error of list_dir is not handled properly, this is a fix.

### CODE REVIEW

File: imjoy/imjoyPluginEngine.py [hunk 1]
```diff
@@ -1014,26 +1014,33 @@ def scandir(path, type=None, recursive=False):
...
+        files_list["children"] = scandir(files_list["path"], type, recursive)
 
-    if sys.platform == "win32" and os.path.abspath(path) == os.path.abspath("/"):
-        files_list["drives"] = get_drives()
+        if sys.platform == "win32" and os.path.abspath(path) == os.path.abspath("/"):
+            files_list["drives"] = get_drives()
 
-    return files_list
+        return files_list
+    except Exception as e:
```
Comment (MartinHjelmare) [inline imjoy/imjoyPluginEngine.py]: Do we need to put the except block down here? What calls can blow up? It's good practice to wrap as little as possible in the try... except. We can eg use try... except... else if needed.

Comment (oeway) [reply imjoy/imjoyPluginEngine.py]: Calling scandir can fail due to non-existing path for example.

Comment (oeway) [reply imjoy/imjoyPluginEngine.py]: Calling scandir can fail due to non-existing path for example.

Comment (MartinHjelmare) [reply imjoy/imjoyPluginEngine.py]: I would only wrap the line I know can error, and if we know the error we should catch that explicitly.

**Pull request merged**